Automattic/isolated-block-editor
# Collab: fix multi line carets

Fixes #59.

## Testing instructions

Run storybook with `yarn storybook` (you need to use a workaround from https://github.com/Automattic/isolated-block-editor/pull/74 temporarily to get it running). Check out the Collaboration -> Components -> Caret -> Multi Line storybook to see the fix in action. You can also use the Default collaboration component, open it in 2 tabs, copy&paste longer text and see if the issue is fixed.

Props @mirka for pairing with me on this and showing me how collab works.
